plugins/inputs/postgresql/postgresql.go

```diff
@@ -15,8 +15,9 @@ import (
 
 type Postgresql struct {
 	Service
-	Databases        []string
-	IgnoredDatabases []string
+	Databases          []string
+	IgnoredDatabases   []string
+	PreparedStatements bool `toml:"prepared-statements"`
```

Comment:
```suggestion
	PreparedStatements bool `toml:"prepared_statements"`
```

Reply:
Could you also copy the sample config into the README.md? Seems to be missing. Thanks!